vultr.php

```diff
@@ -2239,12 +2246,29 @@ private function getTabActions($package, $service, array $post = null, $client =
             $server_details = $this->parseResponse($vultr_api->listBaremetal($params));
         }
 
+        // Get the server ipv6
+        $ip_list = null;
+        if ($service_fields->vultr_enable_ipv6 == 'enable') {
+            $errors = $this->Input->errors();
+            $this->log('api.vultr.com|listIpv6', serialize($params), 'input', true);
+            $ip_list = $this->parseResponse($vultr_api->listIpv6($params));
+
+            if ($errors != $this->Input->errors()) {
```

Comment:
This seems odd. You check whether the errors are the same as any new input errors set from parsing the API response, but then you override them with the original input errors? I'm not sure what the goal is here. Why not merge the two error sets into one?

Reply:
So this is a call that checks for the currently assigned server ipv6 address.  If there are none then the call will return an error.  But we don't actually want to display that error because it is totally expected.  If there were errors before this call though, we still want to show those.  So this says "if we set new errors, ignore them and reset errors to what they were before."  I'll add a comment to clarify.